how-to/customize-workspace/client/src/integrations-shapes.ts

```diff
@@ -56,7 +56,7 @@ export interface IntegrationManager {
 /**
  * Integration provider settings.
  */
-export interface IntegrationProvider {
+export interface IntegrationProviderOptions {
```

Comment:
Change to integration-shapes need copying to the customize-home-templates as well

Reply:
Updated home template examples